src/SFA.DAS.Provider.PR.Web/Controllers/EmployerDetailsController.cs

```diff
@@ -54,4 +61,13 @@ public async Task<IActionResult> Index([FromRoute] int ukprn, [FromRoute] Guid r
 
         return View(model);
     }
+
+    private static bool IsActivePermissionsRequest(Response<GetRequestsByRequestIdResponse> response)
+    {
+        var responseContent = response.GetContent();
```

Comment:
Define the type explicitly here instead of var so we one can see which object it is and can see the definition of it easily. Var can be used where it is clear what the object type is.